package.json

```diff
@@ -34,16 +23,27 @@
     "fsevents",
     "mac"
   ],
-  "author": "Philipp Dunkel <pip@pipobscure.com>",
+  "contributors": [
+    {
+      "name": "Philipp Dunkel",
+      "email": "pip@pipobscure.com"
+    },
+    {
+      "name": "Ben Noordhuis",
+      "email": "info@bnoordhuis.nl"
+    },
+    {
+      "name": "Elan Shankar",
+      "email": "elan.shanker@gmail.com"
+    }
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/strongloop/fsevents/issues"
   },
-  "bundledDependencies": [
-    "node-pre-gyp"
-  ],
   "homepage": "https://github.com/strongloop/fsevents",
   "devDependencies": {
-    "tap": "~0.4.8"
+    "node-gyp": "^3.8.0",
```

Comment:
Don't take a dependency on a specific node-gyp version, that's a huge anti-pattern. Just use the one that's bundled with npm.